src/it/scala/net/snowflake/spark/snowflake/ColumnNameCaseSuite.scala

```diff
@@ -174,7 +174,16 @@ class ColumnNameCaseSuite extends IntegrationSuiteBase {
          |"SF_CONNECTOR_QUERY_ALIAS" ) AS "SUBQUERY_0" ) AS "SUBQUERY_1" WHERE
          |( ( "SUBQUERY_1"."SUBQUERY_1_COL_2" IS NOT NULL ) AND ( "SUBQUERY_1".
          |"SUBQUERY_1_COL_2" = 1 ) )
-         |""".stripMargin.replaceAll("\\s", ""))
+         |""".stripMargin.replaceAll("\\s", "")) ||
+        Utils.getLastSelect
```

Comment:
The spark plan in Spark 3.2 is different comparing with Spark 3.1. So the pushdown query is different. 
This affects a lot of test case.

We want the test case to support both 3.2 and 3.1/3.0. So, I will update the affected test case.
If the actual query is equals to either version, we think the test case to be true.

1. This test case checked the query directly, so I update it directly.
2. In many test cases, `testPushdown()` is used to check pushdown result and query.
    I will create a new test method: `testPushdownMultiplefQueries(queries: Seq[String],...)`
    which will check the last query to be equal to any query in `queries`.